src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/NoopTelemetryModule.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.ApplicationInsights.Extensibility;
+
+namespace Microsoft.Azure.WebJobs.Logging.ApplicationInsights
+{
+    /// <summary>
+    /// Noop telemetry module. Used when customers disables certain modules.
+    /// </summary>
+    internal class NoopTelemetryModule : ITelemetryModule
+    {
+        public void Initialize(TelemetryConfiguration configuration)
+        {
+        }
+    }
+}
```

Comment:
Can you port the `NullTelemetryModule` change from https://github.com/Azure/azure-webjobs-sdk/pull/2073 over to here? We'll get this PR in first...

Reply:
Sure, I was waiting for #2073 to get merged so I can rebase and avoid conflicts. Do you want to merge this one first?

Reply:
Yeah, let's do this one first since it's less impactful.

Reply:
And actually... not to make this too big, but can you just move ApplicationInsights forward to the latest nugets here as well? That way if #2073 slips for any reason, we'll still get that change into this release next week.